pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
   "Programming Language :: Python :: 3.11",
   "License :: OSI Approved :: Apache Software License",
 ]
-dependencies = ["numpy", "protobuf<4", "onnx", "typing_extensions"]
+dependencies = ["numpy", "protobuf<5", "onnx", "typing_extensions"]
```

Comment:
Do we actually need protobuf as a dependency, or do we inherit this from onnx? @gramalingam

Reply:
I removed it